templates/shared/_blender-contact-us-form.html

```diff
@@ -0,0 +1,78 @@
+<section class="p-strip is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h1>{{h1}}</h1>
+      <p>{{intro_text}}</p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <form action="https://pages.ubuntu.com/index.php/leadCapture/save" method="post" id="mktoForm_{{formid}}">
+
+        <fieldset class="u-no-margin--bottom">
+          <h3>Tell us about your project and what problem(s) you’re looking to solve.</h3>
+          <input required id="Company" name="Company" maxlength="255" type="text" class="mktoField mktoRequired" />
```

Comment:
A few things:
- The `h3` should be a `label` (it could have the class `.p-heading--three` if the font needs to be that size, @anasereijo what do you think?)
- The input should be a textarea
- the `id` and `name` attributes have incorrect values

Take a look at the [cloud contact form template](https://github.com/canonical-web-and-design/ubuntu.com/blob/master/templates/shared/_cloud-contact-us-form.html#L53-L54), I think you should be able to use this same label + field